MusyncSavDecode_old.py

```
@@ -1,244 +1,244 @@
+			Unknown0 = ''.join(['%02X' % b for b in self.saveData[4:8]])
+			temp = ''.join(['%02X' % b for b in self.saveData[8:12]])
+			SpeedStall = temp[6:]+temp[4:6]+temp[2:4]+temp[0:2]
+			Unknown1 = ''.join(['%02X' % b for b in self.saveData[12:16]])
+			SyncNumber = str(int.from_bytes(self.saveData[16:20], 'little'))
+			UploadScore = unpack('<f',bytes.fromhex(self.Bytes2HexString(self.saveData[20:24])))[0]
+			PlayCount = int.from_bytes(self.saveData[24:28], 'little')
+
+			if self.saveData[28]==1:
+				statu = "Favo"
+				# FavSong.write(songName[0]+"\n")
+				self.FavSong.append(songName[0])
```

Ruff (0.14.8)

Undefined name `songName`

(F821)